### PR TITLE
Feature/added category and prep cook time

### PR DESCRIPTION
**Pull request by ericmuijs**
This code allows to get prep_time, cook_time and category from schema recipe, if exists.

### CODE REVIEW

File: recipe_scrapers/_schemaorg.py [hunk 1]
```diff
@@ -77,6 +83,18 @@ def get_key_and_minutes(k):
             total_time = sum(times)
         return total_time
 
+    def cook_time(self):
+        if not (self.data.keys() & {"cookTime"}):
```
Comment (jayaddison) [inline recipe_scrapers/_schemaorg.py:87]: This took a few moments to understand - could `if "cooktime" not in self.data.keys()` be a simpler way to express this?

Comment (ericmuijs) [reply recipe_scrapers/_schemaorg.py:87]: Hi Jay, did you check the original code for function total_time in the same file? I copied the same code as used there. I also think this is hard to read.

Comment (ericmuijs) [reply recipe_scrapers/_schemaorg.py:87]: @jayaddison is there something I have to do now? Regards.

**Comment by hhursev** [comment]
Thanks! 🍻

**Pull request merged**